Src/LinearSolvers/MLMG/AMReX_MLLinOp.cpp

```diff
@@ -342,6 +342,7 @@ MLLinOp::make (Vector<Vector<MultiFab> >& mf, int nc, int ng) const
         {
             const auto& ba = amrex::convert(m_grids[alev][mlev], m_ixtype);
             mf[alev][mlev].define(ba, m_dmap[alev][mlev], nc, ng, MFInfo(), *m_factory[alev][mlev]);
+	    mf[alev][mlev].setVal(0.0);
```

Comment:
Do we have to have this setVal here?  Could we do it where make is called?

Reply:
My thought was that it is good practice to instantiate right after definition. However we can certainly instantiate elsewhere if that is the amrex convention or if it would improve performance.

Reply:
Besides performance, it will help debugging.

Reply:
Ok sounds good. I will make the change.